assets/questions/mostCommonChar.js

```diff
@@ -0,0 +1,117 @@
+// Copyright 2017 The TIE Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Question data for Most Common Character.
+ */
+
+globalData.questions['mostCommonCharacter'] = {  // eslint-disable-line dot-notation
+  title: 'Most Common Character',
+  starterCode: {
+    python:
+`def findMostCommonChar(s):
+    return ""
+`
+  },
+  auxiliaryCode: {
+    python:
+`from collections import Counter
+
+class AuxiliaryCode(object):
+    @classmethod
+    def lettersOnly(cls, word):
+      if word == "":
+        return ""
+      word = [w for w in word if w.isalpha()]
+      counter = Counter(word)
+      result = word[0]
+      for w in counter:
+        if counter[w] > counter[result]:
+          result = w
+      return w
+`
+  },
+  tasks: [{
+    instructions: [
+      [
+        "For this problem, we'd like you to write a method to determine ",
+        'the most common character in a string. You will be given a string ',
+        'of ASCII characters, and you may assume that there ',
+        'is only one most common character, ',
+        'and you do not have to consider spaces, for now.'
+      ].join(''),
+      "There's no need to validate that you're always passed a string."
+    ],
+    prerequisiteSkills: ['String Manipulation', 'Arrays'],
+    acquiredSkills: ['String Manipulation'],
+    inputFunctionName: null,
+    outputFunctionName: null,
+    mainFunctionName: 'findMostCommonChar',
+    correctnessTests: [{
+      input: '',
+      allowedOutputs: ['']
+    }, {
+      input: 'apoiuytrewqsdf*&^%$#ba',
+      allowedOutputs: ['a']
+    }, {
+      input: 'aBBB4562873ba',
+      allowedOutputs: ['B']
+    }, {
+      input: 'babaabb',
+      allowedOutputs: ['b']
+    }],
+    buggyOutputTests: [{
+      buggyFunctionName: 'AuxiliaryCode.lettersOnly',
+      messages: [
+        "Try running code on '1600Amphitheatre'. Have you considered digits?",
+        "Did you consider special characters such as '~!?@#' ?",
+        "It looks like you didn't consider all ASCII characters."
```

Comment:
Suggest the following to soften the tone a bit: "It looks like you might not have considered different types of ASCII characters. Check your code to see if it properly handles strings with digits, such as '1600Amphitheatre'. Don't forget to consider special characters too, such as '~!?@#' ?'"